src/components/Common/AnimateSpawn/index.tsx

```diff
@@ -0,0 +1,41 @@
+import { useSpawnAnimation } from "@site/src/utils/use-spawn-animation";
+import { ForwardRefComponent, motion, Variants } from "framer-motion";
+import React from "react";
+
+type Props<A, B> = {
+  el?: ForwardRefComponent<A, B>;
+  children?: React.ReactNode;
+  variants: Variants;
+  className?: string;
+
+  // todo: fix this hack
```

Comment:
is this consciously left in here? There's a few more TODOs in this file

Reply:
Yes, it is functional, we need to release this branch on short notice.